### ISSUE TITLE
Remove all doc references to ol 4.6.5 and replace with latest

### ISSUE THREAD

**Issue by stazrad** [MonsantoCo/ol-kit#223]
A few links are still out of date & pointing to the old version of the docs- check through the `src/` dir and find anything pointing to `4.6.5` and replace it with the latest

**Comment by stazrad** [comment]
@UsmanAhmadSaeed this would be easy 💰

**Comment by LoisGitStuff** [comment]
Hi there, I am pretty new to GitHub, so something easy like this would be perfect for me! 

Could I help with this?

**Comment by stazrad** [comment]
@LoisGitStuff absolutely! We recently upgraded our openlayers dependency from `4.6.5` to the latest and need the links in the documentation to reflect that.
1. fork this repo and checkout the `master` branch.
2. run `npm i` to install all dependencies
3. search in the `src/` directory for any link that goes to the openlayers docs at `4.6.5`
4. replace any of these links with a link to the equivalent doc in [latest openlayers docs](https://openlayers.org/en/latest/apidoc/)
5. to see changes, run `npm run docs` and check the that the links go to latest
6. open a PR into `master`!

**Comment by UsmanAhmadSaeed** [comment]
I believe someone started this! @stazrad thanks for remembering me though! ;-)

**Comment by drodenberg** [comment]
@LoisGitStuff Just wanted to check in and see how this was going and if it was making sense?

**Comment by UsmanAhmadSaeed** [comment]
@LoisGitStuff if you need any guidance, I'd be happy to help as well.

**Comment by LoisGitStuff** [comment]
I'm stuck... I forked and checked out the master branch, but I can't get npm i to work. It gives me an error: npm is not recognized as an internal or external command operable program or batch file.

**Comment by stazrad** [comment]
@LoisGitStuff oh you might have to install node? If you've never used node before you can install it here: https://nodejs.org/en/

**Comment by LoisGitStuff** [comment]
Thank you! After I installed node, I could run npm i.  I found 5 files that contain v.4.6.5. 
What do I change 4.6.5 to?

**Comment by drodenberg** [comment]
So it will just be the latest version of openlayers. Which you should be to replace it with the word `latest` but I would still just make sure that the links work for the openlayers site. So for instance... there's this in the code base in the classes/VectorLayer file: 
`* VectorLayer class extends olLayerVector {@link https://openlayers.org/en/v4.6.5/apidoc/ol.layer.Vector.html}`
and it needs to change to 
`* VectorLayer class extends olLayerVector {@link https://openlayers.org/en/latest/apidoc/module-ol_layer_Vector-VectorLayer.html`

**Comment by stazrad** [comment]
@LoisGitStuff good job getting node working! what @drodenberg said is correct- search for the same name (like `ol.layer.Vector`) in the latest docs: https://openlayers.org/en/latest/apidoc/ & replace the old link (containing `4.6.5`) with the link you find by searching the `latest` docs^

**Comment by LoisGitStuff** [comment]
Hi there, I replaced the outdated '4.6.5' references with the current references. 

![image](https://user-images.githubusercontent.com/68361642/96209091-dae7a280-0f3c-11eb-9413-1de6b8ee446f.png)

But when I ran "npm run docs", I got errors. Are these errors related to something I did wrong? 

![image](https://user-images.githubusercontent.com/68361642/96208785-42e9b900-0f3c-11eb-9485-d800e04f6bee.png)

**Comment by UsmanAhmadSaeed** [comment]
@LoisGitStuff These errors are resulting, because rm is a Linux command and works only on Linux based OSes. I'd suggest you to download and install "Git", open git bash, cd into the ol-kit directory and execute `npm run docs` from git bash instead.
Link for Git Bash -> https://git-scm.com/downloads

**Comment by stazrad** [comment]
@UsmanAhmadSaeed thanks for the help! we don't use linux so I wouldn't even know how to solve that -- @LoisGitStuff you should open a PR with your current changes (4.6.5 replacements) and one of us can do the `npm run docs` command on our machines to get you unblocked.

**Comment by UsmanAhmadSaeed** [comment]
@stazrad No worries. MacOS's terminal/shell is also Linux based I guess, that's why it works on Mac without issues. Windows is the only one giving errors. 😅

**Comment by LoisGitStuff** [comment]
I am using Windows :-)  I already had Git Bash so I ran 'npm run docs'. The site built locally, but I got the following errors. If these errors can be ignored, let me know and I will go ahead and PR the changes. 

![image](https://user-images.githubusercontent.com/68361642/96299977-d82f9080-0fc2-11eb-978a-537e6ed49424.png)

**Comment by stazrad** [comment]
@LoisGitStuff nice! there's a note in there (tho not super obvious) that `"TypeError: Path must be a string." will not effect the docs output` go ahead and open the PR! 🚀

**Comment by LoisGitStuff** [comment]
So I did an add for each file that I updated, then I tried to commit the changes using the following text at the command prompt: 

git commit -m "Fixed out-of-date links to pages in OpenLayers website."

But I got a whole bunch of errors. Are the errors related to the fact that running "npm run docs" changed a bunch of other files that I did not include in the add?  Should I ignore the errors? 

 
![image](https://user-images.githubusercontent.com/68361642/96325989-e8fbf880-0ffa-11eb-9741-d234c6aa864b.png)

**Comment by stazrad** [comment]
@LoisGitStuff oh shoot. That’s the linter. You can commit by passing ‘—no-verify’ at the end of the commit command (same thing for pushing if a test fails)

**Comment by LoisGitStuff** [comment]
The no-verify wanted a comment, so I added  -m "<comment>" to the end of the command, which seemed to work. 

I opened the PR (yay!), but I also wanted to share the messages I got for the Push in case they affect the files that got pushed. 
 
![image](https://user-images.githubusercontent.com/68361642/96338945-5c375600-105f-11eb-895e-cb7c2d30c9a6.png)

Thank you so much for your support and patience!
